float64 start_sec
Note[] song
